lib/types/enums.ts

```diff
@@ -4,6 +4,13 @@ export enum SwapClients {
   RAIDEN,
 }
 
+export enum ReputationEventType {
+  ManualBan = 'manualBan',
```

Comment:
What about adding new events though? When you add a new field to an enum those automatic 0, 1, 2, 3 move accordingly. Adding, removing or just changing the position of a single event is a breaking change that way.

Reply:
That's true. If we add new events to the end of the list of enums then we don't interfere with the existing ones, BUT if we ever delete an event it could mess things up, which I hadn't really considered. I think then maybe the best approach would be to explicitly assign each enum an integer value. `MANUAL_BAN = 0` and so on... That should prevent unintentional breaking changes, but also give us flexibility in terms of being able to rename events plus reducing the amount of data we need to store in the database for each event.

Reply:
I am still in favor of using `MANUAL_BAN = 'manualBan'` instead of a number because I think the few bytes more to store are negligible and if we ever remove an event without adding a new one we skip a number which looks messy. Having to change the `manualBan` is very unlikely.

I will implement it the way you described it but I am not 100% convinced of that approach.

Reply:
Also should the `SwapDealPhase` isn't using UPPER_SNAKE_CASE. Should I update that enum in this PR?

Reply:
So as I see it, it's true that saving bytes in the db is not a big deal. Aside from that the advantage of string-valued enums is that there's no risk of removed events leading to out-of-sequence enum values (0, 1, 3, 4, etc..), which I also don't think would be a deal breaker since the code puts these values behind the scenes but I agree it's not ideal. The advantage of number-valued enums is we can rename the enums internally. I'd guess that we're probably not going to be either renaming reputation events nor removing them all that often, so it's kind of a wash. I'd still lean to using numbers although I suppose either way is fine. Maybe somebody else can chime in.